chore: Drop node 22 (#524)
SAP/ai-sdk: docs-js/overview.mdx

To use the SAP Cloud SDK for AI in a JavaScript / TypeScript application, it is necessary to understand the technical prerequisites and required versions for common dependencies.

> - A project using **Node.js v22+** with **native ESM** support. Node.js v20 reached end-of-life on 2026-04-30 and is no longer supported.

Reviewer: [pp]
This is implicit:
```suggestion
- A project using **Node.js v22+** with **native ESM** support.
```

- A project using **Node.js v22+** with **native ESM** support. Node.js v20 reached end-of-life on 2026-04-30 and is no longer supported.

:::tip Node.js Version Recommendations
Node.js v20 is supported but **approaching end-of-life (EOL).**

> ➡️ We strongly recommend upgrading to **Node.js v22 (Active LTS)** or **v24 (Current)** for long-term support, security updates, and improved ESM/CommonJS interoperability.

Reviewer: [pp]
```
Node.js v22 is supported but **approaching end-of-life (EOL).**
➡️ We strongly recommend upgrading to **Node.js v24 (Active LTS)** or **v26 (Current)** for long-term support and security updates.
```

> If your project is still using CommonJS, you can load this SDK with synchronous `require()` on Node.js 22+:

Reviewer: ```suggestion
If your project is still using CommonJS, you can load this SDK with synchronous `require()`:
```

const { AzureOpenAiChatClient } = require('@sap-ai-sdk/foundation-models');
```

> This relies on Node.js 22's built-in `require(ESM)` support and is considered experimental.

Reviewer: ```suggestion
This relies on Node.js 22's built-in `require(ESM)` support.
Be aware that this is considered experimental for Node.js versions older than v25.4.0.
```
